README.md

```diff
@@ -1,51 +1,43 @@
-
-## How It Works
-- Track Your Periods: Select dates on the calendar, press 'period' and your done!
-- Track Your Ovulation: Select the ovulation date, press 'ovulation' and voila! 
-- View Statistics: Easily check your average cycle length, period length, and upcoming period/ovulation predictions.
-- Manage your symptoms: Add and track custom symptoms to reflect your unique cycle.
-- App settings: Change the app's colors, language, and settings to suit your preferences.
-- - Export and Import Data: Use the export feature to save your data as a JSON file and the import feature to load data back into the app.
-  
-## Who Should Use It?
-Mensinator is perfect for anyone seeking a straightforward and private period tracking solution. Whether you're tracking for health reasons or simply keeping an eye on your cycle, Mensinator makes it easy and secure.
-
-## Get Started with Mensinator
-Ready to start tracking your menstrual cycle with ease and privacy? You can download the Mensinator app from your preferred source:
-
-- **[Google Play](https://play.google.com/store/apps/details?id=com.mensinator.app)**
-- **[IzzyOnDroid](https://apt.izzysoft.de/fdroid/index/apk/com.mensinator.app)**
-- **[FDroid](https://f-droid.org/en/packages/com.mensinator.app/)**
-- Or download the APK directly from our [latest release on GitHub](https://github.com/EmmaTellblom/Mensinator/releases/latest).
-
-Choose the option that works best for you and take control of your cycle tracking with Mensinator!
-
-
-## Contributing
-We welcome contributions to improve Mensinator! If you'd like to contribute, please fork the repository and submit a pull request. For major changes, please open an issue first to discuss your proposed changes. You are also welcome to join our Discord to discuss further or simply to chat with us!
-Join us here; **https://discord.gg/tHA2k3bFRN**
-
-## License
-This project is licensed under the MIT License - see the LICENSE file for details.
+# Mensinator🩸
+Private period tracking with custom symptoms. We don't track your data - you do. Period.
+
+## 🫂 Made for friends by friends
+Originally Emma made Mensinator for her friend with goals of better symptom tracking and privacy respect.  
+Soon, many of us joined and are now actively building the Mensinator together.  
+It is friendly, straightforward, and respects privacy.  
+We wanted an app to use, not an app to sell data to random corporations.
+
+## 💡 Features
+- Period and ovulation tracking,
+- Customizable and colour-coded symptoms,
+- Statistics and cycle prediction,
+- Data export and import for backups or moving to another device,
+- No tracking. Your data is yours. No data is sent anywhere.
+- No bloat. No useless or complex features. No sign-up. No emails.
+
+## 🔮 Future ideas
+- Different calculation types: Choose between regular and luteal prediction modes depending on (ir)regularity of your cycle,
```

Comment:
This should be removed as its already implemented in the app since a few versions back